gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/DistributedRegion.java

```diff
@@ -4865,7 +4865,9 @@ void updateSizeOnRemove(Object key, int oldSize) {
   @Override
   void updateSizeOnClearRegion(int sizeBeforeClear){
     if(!this.reservedTable() && needAccounting()){
-      callback.dropStorageMemory(getFullPath(), getIgnoreBytes());
+      long ignoreBytes = this.isDestroyed() ? getIgnoreBytes() :
+              getIgnoreBytes() + regionOverHead;
+      callback.dropStorageMemory(getFullPath(), ignoreBytes);
```

Comment:
Ok, but don't we require this accounting for LocalRegions too? For example meta-data tables use this (or temporary tables in future).